phpcs.xml

```diff
@@ -9,7 +9,7 @@
 	<exclude-pattern>*/includes/class-connekt-plugin-installer.php</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 	<rule ref="PHPCompatibility"/>
-	<config name="testVersion" value="5.6-"/>
+	<config name="testVersion" value="7.2-"/>
```

Comment:
WordPress requires 7.0 as min, shouldn't we use that?

Reply:
It is the debate of what version makes sense to support. They recommend 7.4, they set a minimum of 7.0. I sort of split the difference.